pkg/scripts/render.go

```diff
@@ -122,7 +122,7 @@ var containerRuntimeTemplates = map[string]string{
 			Restart=always
 			Environment=CONTAINERD_CONFIG=/etc/containerd/config.toml
 			ExecStart=
-			ExecStart=/usr/bin/env PATH=\${TORCX_BINDIR}:\${PATH} \${TORCX_BINDIR}/containerd --config \${CONTAINERD_CONFIG}
+			ExecStart=/usr/bin/env containerd --config \${CONTAINERD_CONFIG}
```

Comment:
```suggestion
			ExecStart=/usr/bin/env PATH=\${TORCX_BINDIR}:\${PATH} containerd --config \${CONTAINERD_CONFIG}
```

please change it to make it backward compatible

Reply:
LTS was also release "just now", so no wander it works too. I means some older images, before the Release Date: Feb 14, 2024 :shrug:

Reply:
Ahh well, the lts release wasnt released yesterday, when i tested it :D But yeah i will just add it, as it shouldnt hurt